bin/parse

```diff
@@ -5,9 +5,8 @@ $:.unshift(File.expand_path("../lib", __dir__))
 require "optparse"
 require "bundler/setup"
 require "regular_expression"
-require "crabstone"
+# require "crabstone"
```

Comment:
I think we want to add Cranelift and compare against manual generation - rather than remove manual generation.

Reply:
yes sorry, I'm not sure why I did this